compiler-plugin-tests/src/test/resources/static_code_analyzer/ballerina_packages/rule1/main.bal

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2024 WSO2 LLC. (https://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/log;
+import ballerina/http;
+import ballerina/file;
```

Comment:
Should we avoid using downstream modules?

Reply:
Fixed in [81d0c7d](https://github.com/ballerina-platform/module-ballerina-log/pull/1175/commits/81d0c7d8d9fa4aa8a0814c2882a1a506e15dd47c)